bitnami/kafka/values.yaml

```
@@ -225,14 +225,16 @@ auth:
   ## | sasl      | Yes (via SASL)                | No                 |
   ## | sasl_tls  | Yes (via SASL)                | Yes                |
   ## @param auth.clientProtocol Authentication protocol for communications with clients. Allowed protocols: `plaintext`, `tls`, `mtls`, `sasl` and `sasl_tls`
+  ## @param auth.externalClientProtocol Authentication protocol for communications with external clients. Defaults to value of `auth.clientProtocol`. Allowed protocols: `plaintext`, `tls`, `mtls`, `sasl` and `sasl_tls`
   ## @param auth.interBrokerProtocol Authentication protocol for inter-broker communications. Allowed protocols: `plaintext`, `tls`, `mtls`, `sasl` and `sasl_tls`
   ##
   clientProtocol: plaintext
+  externalClientProtocol: ""
```

This parameter should have a default protocol (as the rest)

---

It's intentionally set to "" so that it will default to the configured value of clientProtocol. If we specify it here it will break a lot of existing configs.

---

I can see your point... Let's add a warning comment to avoid this to be broken in the future:
```suggestion
  # Note: empty by default for backwards compatibility reasons, find more information at
  # https://github.com/bitnami/charts/pull/8902/
  externalClientProtocol: ""
```